app/views/pages/help.html.erb

```diff
@@ -459,6 +459,87 @@
         </tbody>
       </table>
 
+      <h3 id="nibrs_incident_reports">NIBRS Incident Reports</h3>
```

Comment:
I love love love that you documented this. Documenting the columns like you did is especially helpful.